src/core/core.js

```diff
@@ -226,6 +232,22 @@ export default class Core {
       });
   }
 
+  clearResults () {
+    this.globalStorage.set(StorageKeys.QUERY, null);
```

Comment:
does using globalStorage.delete work here?

Reply:
it does not. currently `delete`s do not emit an update, only `set`s currently.